.storybook/webpack.config.js

```diff
@@ -65,6 +65,21 @@ module.exports = async ( { config } ) => {
 			},
 		],
 		include: path.resolve( __dirname, '../' ),
+	},
+	{
+		test: RegExp( 'node_modules/@material/web/.*\.js' ),
```

Comment:
Note, the published `@material/web@0.1.0-alpha.0` package uses the `??` nullish coalescing operator, necessitating some additional rules to our webpack configs.